### PR TITLE
Build work again on travis 

### PR DESCRIPTION
**Pull request by brodock**
Changed to build with docker. Specs still need to be fixed, but it is correctly building on intended architectures.

I've removed old unsupported ones.

### CODE REVIEW

**Comment by dwmarshall** [comment]
I'll try merging this into what I've got for #124

**Comment by dwmarshall** [comment]
The merge didn't work out correctly. I'll make you a collaborator on my repo in case it is convenient for you to have greater access.

https://travis-ci.org/dwmarshall/ark/builds/107702998

is the merge of #124 and #125

**Comment by brodock** [comment]
@dwmarshall its in an intended state.
here is the summary of the problems that still need some work to be done:

CentOS 6 [see build](https://travis-ci.org/dwmarshall/ark/jobs/107702999#L1331):
> No such file or directory - /bin/tar

Debian 7 [see build](https://travis-ci.org/dwmarshall/ark/jobs/107703000#L1688):
> STDERR: configure: error: cannot find install-sh, install.sh, or shtool in config "."/config

Ubuntu 12.04 [see build](https://travis-ci.org/dwmarshall/ark/jobs/107703001#L2110):
> STDERR: /bin/bash: /usr/local/test_autogen-1/config/missing: No such file or directory

Ubuntu 14.04 [see build](https://travis-ci.org/dwmarshall/ark/jobs/107703003#L1811):
> STDERR: configure: error: cannot find install-sh, install.sh, or shtool in config "."/config

So basically we must make sure tar is present for centos (maybe another recipe to the run_list), and fix autogen, for debian/ubuntu

**Comment by dwmarshall** [comment]
Thank you for the clarification! I'm hoping to have the remaining work done tonight.

**Comment by dwmarshall** [comment]
BTW, the trick for centos is to merge #61 and to add `bzip2` for the list of packages for centos

**Comment by dwmarshall** [comment]
closing in favor of #126, which contains these commits
